egs/wsj/s5/steps/nnet3/multilingual/allocate_multilingual_examples.py

```diff
@@ -95,27 +71,18 @@ def get_args():
         'output-2'.""",
         epilog="Called by steps/nnet3/multilingual/combine_egs.sh")
 
-    parser.add_argument("--samples-per-iter", type=int, default=40000,
+    parser.add_argument("--samples-per-iter", type=int, default=10000,
                         help="The target number of egs in each archive of egs, "
                         "(prior to merging egs). ")
-    parser.add_argument("--num-jobs", type=int, default=20,
-                        help="This can be used for better randomization in distributing "
-                        "examples for different languages across egs.*.scp files, "
-                        "where egs.<job>.*.scp are generated "
-                        "randomly and combined across all jobs in egs.*.scp files.")
-    parser.add_argument("--random-lang", type=str, action=common_lib.StrToBoolAction,
-                        help="If true, egs.ranges.*.txt are generated "
-                        "randomly w.r.t distribution of remaining examples in "
-                        "each language, otherwise it is generated sequentially.",
-                        default=True, choices = ["false", "true"])
-    parser.add_argument("--max-archives", type=int, default=1000,
-                        help="max number of archives used to generate egs.*.scp")
-    parser.add_argument("--seed", type=int, default=1,
-                        help="Seed for random number generator")
-    parser.add_argument("--minibatch-size", type=int, default=512,
-                        help="It is the number of consecutive egs that is taken "
-                        "from each input scp source, and it only affects locality "
-                        "of disk access. This does not have to be actual minibatch size.")
+    parser.add_argument("--num-archives", type=int, default=None,
+                        help="Number of output archives (i.e. output scp files)."
```

Comment:
change this sentence to:
"Number of archives to split the data into.  (Note: in reality they are not "
"archives, only scp files, but we use this notation by analogy with the "
"conventional egs-creating script).  ... "